cmd/minikube/cmd/node_start.go

```diff
@@ -50,7 +50,10 @@ var nodeStartCmd = &cobra.Command{
 		}
 
 		// Start it up baby
-		node.Start(*cc, *n, nil, false)
+		_, err = node.Start(*cc, *n, nil, false)
```

Comment:
would if this node.Start would it exit and not r recreate on failure ?

Reply:
I've fixed it to retry anytime node.Start fails anywhere